filelink/link.py

```diff
@@ -0,0 +1,49 @@
+import platform
+
+
+HARDLINK = 0
+
+
+def _create_windows(source, destination, link_type):
+    """Creates hardlink at destination from source in Windows."""
+
+    if link_type == HARDLINK:
```

Comment:
Don't forget the `else` where the user passes e.g. `3` in which case it currently silently ignores it.